pkgs/top-level/perl-packages.nix

```diff
@@ -11370,6 +11370,18 @@ let
     };
   };
 
+  NetSFTPForeign = buildPerlPackage rec {
+    name = "Net-SFTP-Foreign-1.89";
+    src = fetchurl {
+      url = "mirror://cpan/authors/id/S/SA/SALVA/${name}.tar.gz";
+      sha256 = "9bd33e130581c1fc3eb6108eaf9056c1507428cace04a572f7afe816d83b08a7";
+    };
+    meta = {
+      description = "Secure File Transfer Protocol client";
+      license = stdenv.lib.licenses.unknown;
```

Comment:
`unknown` is not a valid value

Reply:
Sorry I missed that. Not sure why nix-generate-from-cpan missed the license this time. Fixed.